tun/src/tun_async/mod.rs

```diff
@@ -0,0 +1,59 @@
+use crate::TunInterface;
+use std::io;
+use tokio::io::unix::AsyncFd;
+
+pub struct AsyncTunInterface {
```

Comment:
I am okay with the `AsyncTunInterface` name, but there is precedent for naming the sync and async variants the same if we want to do that, like `tun::TunInterface` and `tun::tokio::TunInterface`

one example is reqwest: [`reqwest::blocking::Client`](https://docs.rs/reqwest/latest/reqwest/blocking/struct.Client.html) vs [`reqwest::Client`](https://docs.rs/reqwest/latest/reqwest/struct.Client.html)

Reply:
Hi, I do agree that it would be cleaner on the user side to follow this pattern, and renamed `AsyncTunInterface` to `TunInterface`.